packages/clay-date-picker/src/Hooks.ts

```diff
@@ -145,15 +145,15 @@ function getWeekArray(d: Date, firstDayOfWeek = 0): Month {
 	for (let i = 7 - firstWeek.length; i > 0; i -= 1) {
 		const outsideDate = clone(firstWeek[0].date);
 		outsideDate.setDate(firstWeek[0].date.getDate() - 1);
-		firstWeek.unshift({date: outsideDate, outside: true});
+		firstWeek.unshift({date: outsideDate, outside: 'previous-month'});
```

Comment:
The main reason is to keep the classes semantically correct. If for some reason we decide `next-month-date` should be slightly different from `previous-month-date`, makes it easier to do. Also I was thinking if someone wanted to perform methods when `next-month-date` was clicked, it would be easier to target.

Reply:
Makes sense, maybe this got a little weird due to the use of the `outside` property but as you made a change to separate it in two, maybe this is better because we don’t need to link the class name in the utility and we only define it in the component.